lib/ecto/migrator.ex

```diff
@@ -662,6 +662,8 @@ defmodule Ecto.Migrator do
     migration_source
     |> Enum.flat_map(fn
       directory when is_binary(directory) ->
+        warn_for_misnamed_files(directory)
+
         Path.join([directory, "**", "*.exs"])
```

Comment:
What do you think about changing this wildcard to "*.{exs, ex}" and then we emit the warning on extract_migration_info (and return nil)?

Reply:
Hrm sure also a valid approach, pushed the update!